homeassistant/components/tplink/entity.py

```diff
@@ -68,6 +68,8 @@
     # update
     "current_firmware_version",
     "available_firmware_version",
+    # siren
+    "alarm",
```

Comment:
I'm not 100% sure but this might interfere with https://github.com/home-assistant/core/pull/124930 which we'll want to cherry pick into the beta so I think we should hold off merging this until 124930 is in.